src/tests/Interop/IDynamicInterfaceCastable/Program.cs

```diff
@@ -68,6 +69,12 @@ public static int GetNumberStatic()
             return GetNumberStaticReturnValue;
         }
 
+        public static int GetNumberHelperReturnValue = 4;
+        int GetNumberHelper()
+        {
+            return GetNumberHelperReturnValue;
```

Comment:
This appears incorrect. We expect this call to trigger an exception right? This means we should call `Assert.Fail("<DESCRIPTION OF VIOLATION>");`